invenio/b2share/modules/b2deposit/b2share_model/metadata/eiscat_metadata_config.py

```diff
@@ -0,0 +1,83 @@
+from invenio.ext.sqlalchemy import db
+
+domain = 'EISCAT'
+# display_name = 'European Incoherent Scatter Scientific Association'
+display_name = 'EISCAT'
+table_name = 'EISCAT'
+image = 'domain-eiscat.png'
+kind = 'project'
+domaindesc = 'Incoherent scatter radar data'
+
+# the domain administators can edit even published records
+admin_can_edit_published_record = True
+
+# only the domain administrators can deposit
+depositing_groups = ['eiscat_domain_administrators']
+
+fields = [
+    {
+        'name': 'instrument',
+        'col_type': db.String(256),
+        'display_text': 'Instrument name',
+        'description': 'Instrument name',
+        'required': False
+    },
+    {
+        'name': 'kindat',
```

Comment:
Do we have any meaningful information about this field? What is it? Is it the same as the one below, `kind_of_data_file`?